lib/server/readMetadata.js

```diff
@@ -194,6 +170,8 @@ function processMetadata(file) {
 
 // process metadata for all docs and save into core/metadata.js
 function generateMetadataDocs() {
+  console.log('Generating Metadata for Docs....');
```

Comment:
I think you may not be fully rebased to the latest master. I know this was removed on purpose because the logs were too noisy in https://github.com/facebook/Docusaurus/pull/291

Reply:
removed